admin/openwisp-config/Makefile

```diff
@@ -95,7 +96,24 @@ define Package/openwisp-config/install
 		$(PKG_BUILD_DIR)/openwisp-config/files/sbin/openwisp-get-address.lua \
 		$(1)/usr/sbin/openwisp-get-address
 
-	$(CP) $(PKG_BUILD_DIR)/VERSION $(1)/etc/openwisp/
+	$(INSTALL_BIN) \
+		$(PKG_BUILD_DIR)/openwisp-config/files/sbin/openwisp-get-random-number.lua \
+		$(1)/usr/sbin/openwisp-get-random-number
+
+	$(CP) $(PKG_BUILD_DIR)/VERSION $(1)/usr/lib/openwisp-config/
+endef
+
+# for backward compatibility
+define Package/openwisp-config/postinst
+#!/bin/sh
+if [ ! -L $${IPKG_INSTROOT}/usr/sbin/openwisp_config ]; then
+	ln -s /usr/sbin/openwisp-config $${IPKG_INSTROOT}/usr/sbin/openwisp_config
+fi
+endef
+
+define Package/openwisp-config/postrm
+#!/bin/sh
+rm -f $${IPKG_INSTROOT}/usr/sbin/openwisp_config
 endef
```

Comment:
This will be removed automatically. No needed.

Reply:
The agent is now named `/usr/sbin/openwisp-config`, we create a symbolic link to `/usr/sbin/openwisp_config` for backward compatibility. This postrm hook removes the symbolic link. Was this clear?

Reply:
I know what you would like to achieve, but I am asking why do it in such complicated way.